tests/index.ts

```diff
@@ -0,0 +1 @@
+/// <reference path="../node_modules/@types/jest/index.d.ts" />
```

Comment:
We shouldn't need this should we?

Reply:
idk makes my ide happy. Its in the tests directory so I don't think its a big deal